### PR TITLE
PMM-7827 remove ssh keys from ami

### PR DESCRIPTION
**Pull request by atymchuk**
https://jira.percona.com/browse/PMM-7827

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+- Build Percona Server for MySQL
+
 ```
 make mysql57-ovf
 make mysql57-ami
 ```
-
-
-## License
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FPercona-Lab%2Fpercona-images.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FPercona-Lab%2Fpercona-images?ref=badge_large)
```
Comment (EvgeniyPatlan) [inline README.md:36]: why there is nothing about this change in commit message?

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (atymchuk) [reply README.md:36]: Well, `Update the readme` was the message :) 
But in essence, we don't need this repo to be scanned by Fossa, so I'll remove it altogether!
Done.

**Comment by EvgeniyPatlan** [comment]
@atymchuk where is the test build and tests on new AMI image?

**Review by BupycHuk — APPROVED** [review]

**Comment by EvgeniyPatlan** [comment]
@atymchuk are there any tests?

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]
Where is the test run?

**Pull request closed without merging**